### PR TITLE
Add Health Checks to Monitoring section

### PR DESCRIPTION
**Pull request by r0ckarong**
https://w3.nue.suse.com/~mnapp/2018-09-05/book.caasp.admin/cha.admin.monitoring.html#sec.admin.monitoring.health

### CODE REVIEW

File: xml/admin_monitoring.xml [hunk 1]
```diff
@@ -96,5 +95,435 @@ https://raw.githubusercontent.com/kubernetes/heapster/release-1.3/deploy/kube-co
...
+    <para>
+     In a HA environment you can monitor <literal>kube-apiserver</literal> on
+     <literal>https://<replaceable>MASTER NODE LOADBALANCER</replaceable>:6443/healthz</literal>.
+    </para>
+    <para>
+     If any master node is running correctly you will receive a valid response.
+    </para>
+    <para>
+     This does, however, not mean that all master nodes necessarily work
+     correctly. To ensure that all master nodes work properly the health checks
```
Comment (svenseeberg) [inline xml/admin_monitoring.xml]: Insert comma after "To ensure that all master nodes work properly"

File: xml/admin_monitoring.xml [hunk 2]
```diff
@@ -96,5 +95,435 @@ https://raw.githubusercontent.com/kubernetes/heapster/release-1.3/deploy/kube-co
...
+  <sect2 xml:id="sec.admin.monitoring.health.node">
+   <title>Node Health Checks</title>
+   <para>
+    The basic check if a node is healthy consists of checking if
+    <literal>kubelet</literal> and the CNI (Container Networking Interface) are
+    working properly.
+   </para>
+
+   <sect3 xml:id="sec.admin.monitoring.health.node.kubelet">
+    <title>Check <literal>kubelet</literal> health</title>
```
Comment (svenseeberg) [inline xml/admin_monitoring.xml]: Optionally, this title could use the progressive form to indicate that the user has to / can do something. "Checking kubelet Health"

File: xml/admin_monitoring.xml [hunk 3]
```diff
@@ -96,5 +95,435 @@ https://raw.githubusercontent.com/kubernetes/heapster/release-1.3/deploy/kube-co
...
+    <para>
+     Endpoint: <literal>https://<replaceable>NODE</replaceable>:10250/healthz</literal> (HTTPS)
+    </para>
+<screen>&prompt.user;<command>curl -i https://localhost:10250/healthz</command>
+ok
+    </screen>
+   </sect3>
+
+   <sect3 xml:id="sec.admin.monitoring.health.node.cni">
+    <title>Check <literal>CNI</literal></title>
```
Comment (svenseeberg) [inline xml/admin_monitoring.xml]: Optionally, this title could use the progressive form to indicate that the user has to / can do something. "Checking CNI"

File: xml/admin_monitoring.xml [hunk 4]
```diff
@@ -96,5 +95,435 @@ https://raw.githubusercontent.com/kubernetes/heapster/release-1.3/deploy/kube-co
...
+     will ensure that &kube; won't route traffic to that specific pod until it's
+     considered to be fully functional and running.
+    </para>
+   </sect3>
+  </sect2>
+
+  <sect2 xml:id="sec.admin.monitoring.health.general">
+   <title>General Health Checks</title>
+   <para>
+    It is advisable to apply other best practices from system administration
```
Comment (svenseeberg) [inline xml/admin_monitoring.xml]: Something we discussed once within the team without putting it into the style guide: @cjschroder recommended to use "We [as in SUSE] recommend to ..."

Comment (cjschroder) [reply xml/admin_monitoring.xml]: Yes, I prefer the active to passive voice, "We recommend applying other best practices..."  It's not a big deal, but maybe we should discuss it one of these for the style guide.

**Pull request merged**